### PR TITLE
[Windows][XPU] Disable Kineto PTI on Windows only

### PR DESCRIPTION
**Pull request by min-jean-cho**
Disable Kineto + XPU PTI on Windows only.

### CODE REVIEW

**Review by guangyey — APPROVED** [review]

**Comment by min-jean-cho** [comment]
@malfet, could you please help review this PR? We will disable Kineto + XPU PTI on Windows only until Kineto is functional on Windows.

File: cmake/Dependencies.cmake
Comment (malfet) [inline cmake/Dependencies.cmake:1563]: One in theory can use clang to compile for windows, than `MSVC` would be false, but `WIN32` would be true regardless of what compiler is used 
```suggestion
  if((NOT USE_XPU) OR WIN32)
```

Comment (min-jean-cho) [reply cmake/Dependencies.cmake]: Makes sense, thanks.

**Comment by min-jean-cho** [comment]
@malfet , looks like it requires your approval before merge.

**Review by malfet — APPROVED** [review]

**Pull request closed without merging**
